src/apps/eth/eth_verify.c

```diff
@@ -134,15 +137,26 @@ app_eth_sign_error_t app_eth_verify_standard_transaction(const ETHSignRequest* r
 {
     const app_eth_coin_params_t* params = app_eth_params_get(request->coin);
     if (params == NULL) {
+        // TODO: This isn't technically invalid, it is just not supported
         return APP_ETH_SIGN_ERR_INVALID_INPUT;
     }
-    if (request->value.size == 0) {
-        // Must transfer non-zero value.
+    if (request->data.size == 0 && request->value.size == 0) {
+        // TODO: This isn't technically invalid, it is just not supported
         return APP_ETH_SIGN_ERR_INVALID_INPUT;
     }
     if (request->data.size != 0) {
-        // Standard tx has no data.
-        return APP_ETH_SIGN_ERR_INVALID_INPUT;
+        char hex[sizeof(request->data.bytes) * 2 + 1] = {0};
+        util_uint8_to_hex(request->data.bytes, request->data.size, hex);
+        confirm_params_t cparams = {
+            .title = "ETH TX data\n(hex)",
+            .body = hex,
+            .scrollable = true,
+            .shorten_body = true,
+            .size = request->data.size,
```

Comment:
rename .size to .display_size, at first i thought it was used to limit the length or the body or simlar.

Reply:
done